.github/workflows/awsfulltest.yml

```diff
@@ -15,6 +15,9 @@ jobs:
     steps:
       - name: Launch workflow via tower
         uses: seqeralabs/action-tower-launch@v2
+        # TODO nf-core: You can customise AWS full pipeline tests as required
+        # Add full size test data (but still relatively small datasets for few samples)
+        # on the `test_full.config` test runs with only one set of parameters
```

Comment:
I'd say this has been done already?
```suggestion
```

Reply:
I cannot apply your suggestion 😢 github gives an error and tells me to refresh which doesn't help. I'll wait for a second review and then do this in a PR if necessary